scripts/pkg/build_templates/opensearch-dashboards/deb/debian/postinst

```diff
@@ -29,23 +29,32 @@ if command -v systemd-tmpfiles > /dev/null; then
     systemd-tmpfiles --create opensearch-dashboards.conf
 fi
 
-# Messages
-echo "### NOT starting on installation, please execute the following statements to configure opensearch-dashboards service to start automatically using systemd"
-echo " sudo systemctl daemon-reload"
-echo " sudo systemctl enable opensearch-dashboards.service"
-echo "### You can start opensearch-dashboards service by executing"
-echo " sudo systemctl start opensearch-dashboards.service"
+# Restart the service after an upgrade
+if [ "$1" == "configure" ] && [ "$2" != "" ]; then
+    if command -v systemctl >/dev/null && systemctl is-enabled opensearch-dashboards.service >/dev/null; then
+        echo "Restarting opensearch-dashboards.service after upgrade"
+        systemctl start opensearch-dashboards.service
```

Comment:
Will `restart` here better than `start`?

Reply:
As of right now, it wouldn't make a difference if "systemctl start" or "systemctl restart" is called, because on a package upgrade the services are stopped in prerm/preinst (e. g. on Upgrade from 2.11.1 to 2.12.0, dpkg first calls the prerm script of version 2.11.1, then the preinst of version 2.12.0 and then postinst of version 2.12.0).

So if you want to change it to "restart" it's ok by me, but it should be known, that due to the other maintainer scripts, it provides the same functionality as "start".